catalog/imaging/imaging-plugin-nitf/src/main/java/org/codice/alliance/plugin/nitf/NitfPostIngestPlugin.java

```diff
@@ -243,36 +288,32 @@ private BufferedImage renderImageUsingOriginalDataModel(ContentItem contentItem)
   }
 
   private BufferedImage render(
-      ContentItem contentItem,
+      InputStream inputStream,
       Function<Pair<ImageSegment, NitfRenderer>, BufferedImage> imageSegmentFunction)
-      throws IOException, ParseException, NitfFormatException {
+      throws NitfFormatException {
 
     final ThreadLocal<BufferedImage> bufferedImage = new ThreadLocal<>();
 
-    if (contentItem != null) {
-      InputStream inputStream = contentItem.getInputStream();
-
-      if (inputStream != null) {
-        try {
-          NitfRenderer renderer = getNitfRenderer();
-
-          new NitfParserInputFlowImpl()
-              .inputStream(inputStream)
-              .allData()
-              .forEachImageSegment(
-                  segment -> {
-                    if (bufferedImage.get() == null) {
-                      BufferedImage bi =
-                          imageSegmentFunction.apply(new ImmutablePair<>(segment, renderer));
-                      if (bi != null) {
-                        bufferedImage.set(bi);
-                      }
+    if (inputStream != null) {
+      try {
+        NitfRenderer renderer = getNitfRenderer();
+
+        new NitfParserInputFlowImpl()
+            .inputStream(inputStream)
+            .allData()
+            .forEachImageSegment(
+                segment -> {
+                  if (bufferedImage.get() == null) {
+                    BufferedImage bi =
+                        imageSegmentFunction.apply(new ImmutablePair<>(segment, renderer));
+                    if (bi != null) {
+                      bufferedImage.set(bi);
                     }
-                  })
-              .end();
-        } finally {
-          IOUtils.closeQuietly(inputStream);
-        }
+                  }
+                })
+            .end();
+      } finally {
```

Comment:
:question: Is is safe to use IOUtils.closeQuietly() without first trying to close the inputstream in the try and handling any exceptions in the catch? All exceptions are ignored in IOUtils.closeQuietly(). https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/IOUtils.html#closeQuietly(java.io.InputStream)